docs/source/technical/vigasp.md

```diff
@@ -0,0 +1,20 @@
+# Overview: 
+VIGAS/P stands for Veteriary Institute Genome Analysis System/Platform. VIGASP uses IRIDA for data management, data access control, analysis and storing the results. IRIDA uses Galaxy to execute the pipelines. 
+
+# Accessing VIGAS and logging in
+VIGAS is accessible only from official VI Laptops or through VI-VPN. 
```

Comment:
stationary computers and VI laptops with "insert the thing you should have for vpn" or through standalone VI-VPN.